modules/local/map_to_tsv.nf

```diff
@@ -23,11 +24,21 @@ process MAP_TO_TSV {
     def delimiter = '\t'
     output_place = task.workDir.resolve(output_file)
 
-    output_place.withWriter{ writer ->
+    metadata = [metadata_headers] + metadata_rows
+    metadata = metadata.transpose()
+    
+    for (int i in (metadata.size()-1)..0) { // Reverse order
+        if (metadata[i][1..-1].every {it == ""}) { // Every except for header
```

Comment:
Could you include a check for null strings in addition to empty strings (e.g, `it == null || it == ""`)

Reply:
Added in 5695b445a1a8fdba6248e855f593cf73abd99e03